Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"name\": \"urlshorteningservice\",\r\n  \"version\": \"1.0.0\",\r\n  \"description\": \"\",\r\n  \"main\": \"index.js\",\r\n  \"scripts\": {\r\n    \"test\": \"echo \\\"Error: no test specified\\\" && exit 1\",\r\n    \"create_docker_image\": \"docker build -t urlshorteningservice .\"\r\n  },\r\n  \"author\": \"\",\r\n  \"license\": \"ISC\",\r\n  \"dependencies\": {\r\n    \"body-parser\": \"^1.19.0\",\r\n    \"express\": \"^4.17.1\",\r\n    \"nodemon\": \"^2.0.12\"\r\n  }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package.json b/package.json
--- a/package.json	(revision 55bcc30225eb639265a01e603bd11985ccbd3521)
+++ b/package.json	(date 1632173512198)
@@ -14,4 +14,4 @@
     "express": "^4.17.1",
     "nodemon": "^2.0.12"
   }
-}
+}
\ No newline at end of file
